elements/rh-navigation-primary/rh-navigation-primary-item.css

```diff
@@ -0,0 +1,460 @@
+
+  display: block;
+  height: 100%;
+  width: 100%;
+  justify-items: center;
+  align-content: center;
+
+  details {
+    width: 100%;
+    height: 100%;
+    justify-items: center;
+    align-content: center;
+
+    &[open] {
+      color: var(--rh-color-text-primary);
+
+      /* stylelint-disable-next-line selector-pseudo-element-no-unknown */
+      &::details-content {
+        display: contents;
+
+        & > * {
+          box-sizing: border-box;
+        }
+      }
+
+      @container (min-width: 1200px) {
+        border-block-start-color: var(--rh-color-accent-brand);
+      }
+
+      & summary {
+        &:focus-visible {
+          outline-color: var(--rh-color-interactive-primary-default);
+        }
+
+        & rh-icon[icon='caret-down'] {
+          rotate: 180deg;
+        }
+      }
+    }
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  summary {
+    --rh-icon-size: 18px /* non standard value */;
+    --rh-avatar-size: var(--rh-size-icon-02, 24px);
+
+    list-style: none;
+    display: flex;
+    gap: var(--rh-space-md, 8px);
+    color: var(--rh-color-text-primary) !important;
+    text-decoration: none !important;
+    text-align: center;
+    text-wrap: nowrap;
+    place-items: center;
+    width: fill-available;
+    width: -webkit-fill-available;
+    width: -moz-available;
+    height: 100%;
+    padding-inline: calc(var(--rh-space-sm, 6px) + var(--rh-space-xs, 4px));
+    cursor: pointer;
+
+    /* stylelint-disable-next-line no-descending-specificity */
+    &:is(:focus-visible, :hover:focus-visible) {
+      outline-offset: -2px;
+      outline:
+        var(--rh-border-width-md, 2px)
+        solid
+        var(--rh-color-interactive-primary-default);
+      border-radius: var(--rh-border-radius-default, 3px);
+    }
+
+    &::marker,
+    &::-webkit-details-marker {
+      display: none;
+    }
+
+    & rh-icon {
+      aspect-ratio: unset;
+
+      /* stylelint-disable-next-line no-descending-specificity */
+      &[icon='caret-down'] {
+        color: var(--rh-color-icon-subtle);
+        transition: rotate 0.2s;
+      }
+    }
+
+    ::slotted(rh-icon) {
+      /* need 24px total icons 18px + 3px margin * 2, avatar is 24px */
+      margin-block: calc(var(--rh-space-sm, 6px) / 2);
+    }
+  }
+
+  #details-content {
+    width: fill-available;
+    width: -webkit-fill-available;
+    width: -moz-available;
+
+    @container (min-width: 1200px) {
+      position: absolute;
+      inset-block-start: 100%;
+      inset-inline-start: 0;
+      z-index: -1;
+      width: 100%;
+      background-color: var(--_background-color);
+      box-shadow: 0 2px 4px 0 var(--_box-shadow-color);
+      border-end-end-radius: calc(var(--rh-border-radius-default, 3px) * 2);
+      border-end-start-radius: calc(var(--rh-border-radius-default, 3px) * 2);
+    }
+  }
+
+  &.standalone {
+    details {
+      display: contents;
+
+      & > * {
```

Comment:
this and the following rules are violating no-descending specificity.

Nesting is great but in this file it's working against readability, please rework this sheet so that it reads top-to-bottom

Reply:
I agree with you this file needs to be revisited; with the number of changes to the design, it has gotten unwieldy in a lot of ways.